tools/tox.ini

```diff
@@ -23,8 +23,8 @@ deps = -rrequirements_flake8.txt
 commands = flake8 --append-config={toxinidir}/py36-flake8.ini {posargs}
 
 [testenv:py36-mypy]
+basepython=python3
```

Comment:
We probably shouldn't have this; probably should just add a `[py37-mypy]` section so this can be run locally under Py3.7 (likewise elsewhere).

Reply:
Add 3.5 as well, please

Reply:
https://github.com/tox-dev/tox/issues/189 would make this so much nicer :(